src/main/scala/io/iohk/ethereum/domain/SignedTransaction.scala

```diff
@@ -17,42 +19,61 @@ object SignedTransaction {
   val LastByteOfAddress: Int = FirstByteOfAddress + AddressLength
   val negativePointSign = 27
   val positivePointSign = 28
-  val allowedSignValues: Set[Int] = Set(negativePointSign, positivePointSign)
-
 }
 
 case class SignedTransaction(
   tx: Transaction,
-  pointSign: Byte, //v - 27 or 28 according to yellow paper, but it is 37 and 38 in ETH
+  pointSign: Byte, //v
   signatureRandom: ByteString, //r
   signature: ByteString /*s*/) {
 
   import SignedTransaction._
 
-  lazy val bytesToSign: Array[Byte] =
+  lazy val bytesToSign: Array[Byte] = if (pointSign == negativePointSign || pointSign == positivePointSign) {
+    //global transaction
+    crypto.sha3(
+      rlpEncode(RLPList(
+        tx.nonce,
+        tx.gasPrice,
+        tx.gasLimit,
+        tx.receivingAddress.toArray,
+        tx.value,
+        tx.payload)))
+  } else {
+    //chain specific transaction
```

Comment:
global is transaction that do not have chain specific point sign / signature which means that it can be part of block in any chain (and can be replayed on any chain), chain-specific is transaction that is signed in a way that includes chain information so it can be part of block only on single chain
(terminology is from parity client)